cl/users/templates/profile/webhooks_logs.html

```diff
@@ -0,0 +1,69 @@
+{% extends "profile/webhooks_base.html" %}
+{% load static %}
+{% load waffle_tags %}
+{% load humanize %}
+{% block title %} {{ page_title }} &ndash; CourtListener.com{% endblock %}
+
+{% block content %}
+{# Navigation Tabs #}
+  {% include "includes/developer-tabs.html" %}
+  {% flag "user-webhooks" %}
+    <div class="hidden-xs col-sm-1 col-md-2"></div>
+    <div class="col-xs-12 col-sm-10 col-md-8 text-center">
+      <div class="row text-left">
+        <div class="col-xs-12">
+          <div class="panel panel-default webhooks">
+            <div class="panel-heading">
+              <div class="row">
+               {% include "includes/webhooks-header.html" %}
+              </div>
+             <div data-loading="block" class="row">
+                <div class="text-center">
+                  <i class="fa fa-spinner fa-spin"></i> Loading
+                </div>
+             </div>
+            </div>
+            <div class="panel-body">
+              <div class="table-responsive">
+                <table class="table"
+                  hx-ext="loading-states"
+                  data-loading-class="hidden"
+                  >
+                    <thead>
+                    <tr class="active">
+                      <th>Event&nbsp;Type</th>
+                      <th>Event Status</th>
+                      <th>Status code</th>
+                      <th>Date created</th>
+                      {% if sub_page == "logs" %}
+                        <th>Next attempt</th>
+                      {% endif %}
+                      <th></th>
+                    </tr>
+                    </thead>
+                    <tr id="webhook_list_loader" class="hidden">
+                      <td class="text-center" colspan="4">
+                        <i class="fa fa-spinner fa-spin"></i> Loading
+                      </td>
+                    </tr>
+                    <tbody
+                      id="webhook_table_body"
+                      hx-trigger="load, webhookLogsListChanged from:body"
+                      hx-get="{% url 'webhook_events-list' format="html" %}?debug={{debug}}"
+                      data-loading-target="#add-webhook"
+                      data-loading-class="hidden"
+                      hx-target="this"
+                      >
+                    </tbody>
+                  </table>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      <p>Webhooks enable bi-directional APIs by sending data to your server whenever events happen on ours. Use webhooks to get the fastest updates &mdash; without polling.</p>
+      <p class="v-offset-below-2"><a href="{% url "api_index" %}#webhooks" class="btn btn-default btn-lg">Read the Webhook Documentation</a></p>
+    </div>
+  {% endflag %}
```

Comment:
DRY, please.

Reply:
Yes, I moved all this common code to webhooks_base.html so it's not repeated.